docs/docs/stylesheets/extra.css

```diff
@@ -0,0 +1,33 @@
+/* Custom styles for logo */
+.md-logo {
+    width: auto !important;
```

Comment:
Avoid !important whenever you can and make sure this file doesn't break existing style.

Reply:
yeah doesn't break any existing style actually it just makes logo bigger
<img width="998" alt="Screenshot 2024-11-06 at 5 19 54 PM" src="https://github.com/user-attachments/assets/30538a0f-a3c5-4927-a736-46c0c59fbd44">

Reply:
@okhat has told me to do, this way actually for better design